docker-compose/docker-compose.dev.mac.sample.yml

```diff
@@ -4,11 +4,11 @@ services:
   phpfpm:
     volumes: &appvolumes-mac
       - workspace:/var/www/html
-      - ./app:/var/www/html/app:delegated
```

Comment:
Hi! @arendarenko this change is probably going to break the mac setup if you don't use mutagen. Maybe I'm not uderstanding it in the right way, could you explain me this change? Thank you!

Reply:
Maybe it's better to fully switch to mutagen instead or having an option?

Reply:
I dont know if it's a good option because you are making this project dependant on mutagen, and we're forcing ppl to install mutagen in order to be able to use this tool!

Could be great making it optional!

Reply:
@danielozano i think it’s better to add one more tool for Mac OS in order to get much better experience.